src/main.rs

```diff
@@ -52,6 +52,28 @@ fn main() -> PassResult<()> {
                 }
                 return Ok(());
             }
+            cmdline::Subcommand::Format(cmdline::Format { file }) => {
+                let mut ws = frontend::Workspace::construct(
+                    &Some(file.clone()),
+                    &opts.lib_path,
+                )?;
+                let imports = ws.original_imports.drain(..).collect_vec();
```

Comment:
It seems like you're trying to remove the imports from the workspace and keep them in a vector. In that case, it's easier to use `std::mem::take` instead of draining the vector because the latter requires allocating a whole new vector and adding all the elements.

Reply:
```rust
let imports = std::mem::take(&mut ws.original_imports);
```